src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -153,13 +154,23 @@ private static void setupMouseHandler(Application application) {
         new ClicksConverter(application.getFreePanel());
     }
 
+    public static void setup(Application application) {
+        FeatureManager.registerFeature(new DriverFeature());
+        FeatureManager.registerFeature(new WorkspaceFeature());
+        FeatureManager.registerFeature(new DrawerFeature());
+        FeatureManager.registerFeature(new CommandsFeature());
+
+        FeatureManager.initializeAll(application);
+    }
+
     /**
      * Launch the application.
      */
     public static void main(String[] args) {
         EventQueue.invokeLater(new Runnable() {
             public void run() {
                 Application app = new Application("Jobs 2D");
+                setup(app);
                 DrawerFeature.setupDrawerPlugin(app);
                 CommandsFeature.setupCommandManager();
```

Comment:
It seems that *Feature.setup*(...) calls below are redundant, as they are already handled by InitializeAll.